release notes/v0.45.0.md

```diff
@@ -0,0 +1,168 @@
+* [browser#859](https://github.com/grafana/xk6-browser/pull/859) Fix remote object parsing when subtype is `null`.
+
+## Maintenance and internal improvements
+
+- [#2091](https://github.com/grafana/k6/pull/2991) Refactor JS modules system so that is usable in tests. Which allowed the enablement of the tc39 tests for modules [#3040](https://github.com/grafana/k6/pull/3040).
+- [#3025](https://github.com/grafana/k6/pull/3025) Internally stop referring to afero and use an internal package to do all file system interaction. That package still uses afero.
+- [#3036](https://github.com/grafana/k6/pull/3036) and [#3053](https://github.com/grafana/k6/pull/3053) Add options to `scenarios` for usage by browser module. 
+- [#3064](https://github.com/grafana/k6/pull/3064), [#3070](https://github.com/grafana/k6/pull/3070),  [#3075](https://github.com/grafana/k6/pull/3075) and [#3106](https://github.com/grafana/k6/pull/3106) Go dependancies updates. 
+- [#3067](https://github.com/grafana/k6/pull/3067) Add method to retrive all registered metrics.
+- [#3068](https://github.com/grafana/k6/pull/3068) Add metric Sink constructor.
+- [#3078](https://github.com/grafana/k6/pull/3078) Pinning to Alpine 3.17 in Docker builder image. Thank you, @arukiidou 🙇.
+- [#3086](https://github.com/grafana/k6/pull/3086) Fix downloading `.golangci.yml` for PRs from forks. 
+- [#3088](https://github.com/grafana/k6/pull/3088) Make TestDNSResolver less flaky. 
+- [#3094](https://github.com/grafana/k6/pull/3094) Fix example from the run command. Thanks to @rocktwotj 🙇.
+- [#3095](https://github.com/grafana/k6/pull/3095) Maintenance update of `.golangci.yml`.
+- [#3103](https://github.com/grafana/k6/pull/3103) Fix lint and logical issues in `k6/data` module tests.
+- [#3045](https://github.com/grafana/k6/pull/3045), [#3049](https://github.com/grafana/k6/pull/3049), [#3073](https://github.com/grafana/k6/pull/3073) and [#3044](https://github.com/grafana/k6/pull/3044) New issues are now automatically assigned to maintainers, to improve response time on issues. Both new issue and new PR assignments are now not using external actions.
+- [#3109](https://github.com/grafana/k6/pull/3109) Add a way to get the cloudapi Client's base URL. Thanks to @yorugac 🙇.
+## Roadmap
+
+There is now a [public Roadmap document](https://github.com/grafana/k6/blob/master/ROADMAP.md) in the main repo added in [#3028](https://github.com/grafana/k6/pull/3028)
+
+The idea is that we will keep it updated and current or future users will be able to look at it to see on what bigger projects the team is working on.
+
+### Cloud output v2
+
+Work on a new version of the cloud output has been ongoing over this cycle.
+
+While functionally it is now mostly complete, we feel like more testing is still needed and some smaller issues need to be ironed out.
+
+Over the next cycle we will be testing it internally and coming v0.46.0 it will be generally available. With the idea of becoming the default.
```

Comment:
We should clarify what will happen to the current version.

```suggestion
Over the next cycle we will be testing it internally and coming v0.46.0 it will be generally available, with the goal of it becoming the default Cloud output. It will still be possible to use the current version via an option, but we plan to gradually deprecate it.
```

@codebien Is this accurate?

Reply:
My interpretation is that it means after we've changed the default option, so yes, it is accurate. Because it will be possible to fallback on v1 until we merge https://github.com/grafana/k6/pull/3092 (targeted as v0.48.0).

Reply:
OK. I changed the first sentence to "Over the next cycle we will be testing it internally, and in v0.46.0 it will be generally available as the default Cloud output.".